nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -14,7 +14,7 @@
     <license type="file">LICENSE</license>
     <projectUrl>https://github.com/microsoft/cswinrt</projectUrl>
     <dependencies>
-      <group targetFramework=".NET5.0" />
+      <group targetFramework=".NET6.0" />
```

Comment:
There is a mention of .NET 5 above in the description.  Probably want to update that.